src/main/java/sg/ncl/MainController.java

```diff
@@ -2089,7 +2089,7 @@ private String abc(@PathVariable String teamName, @PathVariable String expId, Re
             } else {
                 // everything ok
                 log.info("stop experiment success for Team: {}, Exp: {}", teamName, expId);
-                redirectAttributes.addFlashAttribute("exp_message", "Team: " + teamName + " has stopped Exp: " + realization.getExperimentName());
+                redirectAttributes.addFlashAttribute("exp_message", "Exp: " + realization.getExperimentName() + " in Team: " + teamName + " is stopping. Please reload this page in a few minutes.");
```

Comment:
Experiment xx in team yy is stopping. Please refresh this page in a few minutes.